FloripaBus/Repositories/IRouteRepository.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using Newtonsoft.Json;
+using System.Net;
+
+namespace FloripaBus

```

Comment:
In a lot of cases we have only one implementation for a interface. We do it to avoid coupling between the layers of the architecture for example. When that is the case I like keeping both the Interface and the Class in the same file as it's not a big problem and it makes easier to work on.